bindings/c/Cargo.toml

```diff
@@ -16,10 +16,17 @@
 # under the License.
 
 [package]
-edition = "2021"
 name = "opendal-c"
 version = "0.1.0"
```

Comment:
May or may not we do a simultaneous release. Currently, I leave all `version`s defined in the crate instead of inheriting from the root.

Reply:
All crates, except for `opendal-c`, will have the same version.